### PR TITLE
Layout stage before js

### PR DESCRIPTION
**Pull request by harrycoulton**
Hopefully this is all legit and is awaiting further work. This is the final stage of my layout phase before I go onto adding content and styling it all up ( I think ).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,34 +8,190 @@
     <title>index</title>
     <link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (ashleycoles) [inline index.html:9]: I'd suggest including normalize.css here, above your CSS.

File: index.html [hunk 2]
```diff
@@ -8,34 +8,190 @@
...
+                    <i class="fab fa-linkedin"></i>
+                </a>
+            </div>
+        </div>
+        <div class="contact-area-line">
+            <div class="contact-area-item icon">
+                <i class="fas fa-phone"></i>
+            </div>
+            <div class="contact-area-item contact-area-text">
+                +4414677491
```
Comment (ashleycoles) [inline index.html]: I'd suggest using an A tag for the phone number, using href="tel:#####". This automatically dials the number on mobile devices, or opens skype etc.

File: index.html [hunk 3]
```diff
@@ -8,34 +8,190 @@
...
+            </div>
+        </div>
+        <div class="contact-area-line">
+            <div class="contact-area-item icon">
+                <i class="far fa-envelope"></i>
+            </div>
+            <div class="contact-area-item contact-area-text">
+                hfcoulton93@gmail.com
```
Comment (ashleycoles) [inline index.html]: Would also suggest using an a tag with href="mailto:test@test.com". This automatically opens email clients.

File: index.html [hunk 4]
```diff
@@ -8,34 +8,190 @@
...
     </div>
 
+    <div class="about-me-navbar-bumper-container" id="about-me">
+        <div class="navbar-bumper-about-me">
+        </div>
+        <div class="about-me">
+            <div class="face-name-container">
+                <div class="face-hex-wrapper">
+                    <svg class="face-hex" width="175" height="200">
+                        <polyline id="hexagon" points="87,0 174,50 174,150 87,200 0,150 0,50 87,0"/>
```
Comment (ashleycoles) [inline index.html]: An ID should only be used once on a page, it should be unique. These should be changes to classes and the css should be updated to reflect that.

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (harrycoulton) [reply index.html:9]: done

File: index.html (hunk 2 above)
Comment (harrycoulton) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (harrycoulton) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (harrycoulton) [reply index.html]: done

**Pull request merged**